README.md

```diff
@@ -165,7 +165,7 @@ If you like this project give a ⭐️.
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank Microverse.
+I would like to thank Mr Lucas Erkna for his unbiased help to me in realising this project.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] In the README file, the **Video Project Presentation**  section contains a link to a project presentation that is completely different from your current project. I suggest you get rid of the header or include a presentation of this project in the section. This way, your README file would be more descriptive.

Reply:
Thank you for making me to notice this mistake